iceoryx_utils/include/iceoryx_utils/internal/cxx/vector.inl

```diff
@@ -155,11 +155,11 @@ inline uint64_t vector<T, Capacity>::capacity() const
 template <typename T, uint64_t Capacity>
 inline void vector<T, Capacity>::clear()
 {
-    for (uint64_t i = 0u; i < m_size; ++i)
+    for (uint64_t i = m_size; i != 0U; --i)
```

Comment:
Please update the copyright header so that there is a separate line for each copyright owner.